README.md

```diff
@@ -86,7 +88,7 @@ Each slide type also accepts some type-specific properties:
 - `<default-slide>` -- titles, bullets, and arbitrary content
   - `slide.pre`, `slide.title`, and `slide.sub` create the title block
   - `slide.md`, and `slide.webc` allow for arbitrary content
-  - `slide.background`, `slide.color`, & `slide.mode` CSS values
+  - `slide.background-color`, `slide.color`, & `slide.mode` CSS values
```

Comment:
The `slide.background` is a property that I set in slides to change the background, often using gradients. Even though the default value is a color, this property needs to allow full background shorthand. (applies also to the other places this is used)